news/2019-04-15-featured-artist-updates-dfs.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+title: "Featured Artist Updates: Dictate, Fractal Dreamers & sakuraburst"
+date: 2019-04-15 08:00:00 +0000
+---
+
+We're getting down to earth with some brand new tracks from artists you know and love - namely the Monolithia album from Fractal Dreamers, two new tracks from Dictate, and six songs from sakuraburst's HARPSINGER SINGLES plus more!
```

Comment:
"six songs from sakuraburst's HARPSINGER SINGLES plus more" is kind of misleading because it sounds like you mean six from this album and then more, when really you mean six in total including this album

Reply:
removed the singles listing to make it more concise